test/plugins/features/movies/index.test.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
+const Bluebird = require('bluebird');
+
+const Knex = require('../../../../lib/libraries/knex');
```

Comment:
Since `Knex` and `Movies` are both relative `require`s, they should be in the same block (no new lines between them) and with their equal signs aligned.